From e4d091c594d1b5791f52c9249abd80fd6706e5ee Mon Sep 17 00:00:00 2001
From: Michael Orlitzky <michael@orlitzky.com>
Date: Sun, 28 Jul 2024 08:00:38 -0400
Subject: [PATCH] libvbr/vbr.c: include <stdio.h> for vsnprintf()

This fixes an implicit declaration error with newer compilers and on
musl where stdio.h does not incidentally get included by some other
header.

Bug: https://bugs.gentoo.org/936591
---
 libvbr/vbr.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/libvbr/vbr.c b/libvbr/vbr.c
index cb9124d7..aadfbd03 100644
--- a/libvbr/vbr.c
+++ b/libvbr/vbr.c
@@ -18,6 +18,7 @@
 #include <arpa/inet.h>
 #include <arpa/nameser.h>
 #include <netdb.h>
+#include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
 #include <errno.h>
